src/Uno.Toolkit.Skia.WinUI/Controls/Shadows/ShadowContainer.Paint.cs

```diff
@@ -175,6 +184,24 @@ public void ClipToContent(ShadowPaintState state, SKCanvas canvas)
 			canvas.ClipRoundRect(GetContentShape(state), antialias: true);
 		}
 
+		public static bool IsCornerRadiusProportional(CornerRadius CornerRadius)
```

Comment:
move IsCornerRadiusProportional + GetPointsFromCornerRadius above ClipToContent
helper methods should be grouped together
also they are only used within this class, so you can just mark them as `private`
maybe consider dropping `static`, so you can just access local property

Reply:
Done.